.gitignore

```diff
@@ -88,3 +88,5 @@ yarn-error.log
 .devcontainer
 
 src/images/infrastructure_screenshot_full_sonarqube-dashboard.webp
+src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx
+src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx
```

Comment:
@nwhite222 I'm not sure if these code lines are supposed to be here - what was this for again?

Reply:
I'm also not sure what happened here, but I'm going to try deleting them.